detox/e2e/support/ui/screen/create_or_edit_channel.ts

```diff
@@ -59,6 +59,8 @@ class CreateOrEditChannelScreen {
 
     openEditChannel = async () => {
         // # Open edit channel screen
+        await ChannelInfoScreen.scrollView.tap({x: 1, y: 1});
+        await ChannelInfoScreen.scrollView.scroll(100, 'down');
```

Comment:
Why is this needed? Shouldn't it open the edit screen just right without this?

Reply:
in the smaller 15 Pro, the View Info >  Edit channel option is in bottom and we had to scroll and access.